bootstrap/bootstrap.go

```diff
@@ -1532,9 +1423,9 @@ func (b *Bootstrap) runPreCommandHooks(ctx context.Context) error {
 
 // runCommand runs the command and adds tracing spans.
 func (b *Bootstrap) runCommand(ctx context.Context) error {
-	span, ctx := tracetools.StartSpanFromContext(ctx, "command")
+	span, ctx := tracetools.StartSpanFromContext(ctx, "command", b.Config.TracingBackend)
 	var err error
-	defer func() { tracetools.FinishWithError(span, err) }()
+	defer func() { span.FinishWithError(err) }()
```

Comment:
All of these `defer func() { ... }` calls can be changed to 
```suggestion
	defer span.FinishWithError(err)

Reply:
I'm not sure about that. The arguments to `defer` are evaluated at the `defer`. https://go.dev/blog/defer-panic-and-recover

The existing spelling is a closure that refers to the variable `err`.

Reply:
ah! you're right, good call